experiment/kind-e2e.sh

```diff
@@ -67,6 +67,9 @@ bazel build //cmd/kubectl //test/e2e:e2e.test //vendor/github.com/onsi/ginkgo/gi
 mkdir -p "_output/bin/"
 cp bazel-bin/test/e2e/e2e.test "_output/bin/"
 
+# release some memory after building
+sync || true
+echo 1 > /proc/sys/vm/drop_caches || true
```

Comment:
This is based on kubetest, see `util.flushMem()`, which I setup for the same reason for our standard builds.